alert-common/src/main/java/com/synopsys/integration/alert/common/security/CertificateUtility.java

```diff
@@ -145,6 +145,10 @@ public File getAndValidateTrustStoreFile() throws AlertConfigurationException {
         }
     }
 
+    public void validateCertificateContent(CustomCertificateModel customCertificateModel) throws AlertException {
```

Comment:
This method and the method below this both take a customCertificateModel. How does a user of the API know which validate to call?

Reply:
The method below is private so they wouldn't know it exists. I can change the public method to validateCertificateContents.  I am just validating that the content is a in fact a certificate we can parse and potentially load.

Reply:
Ah, I see. I would still prefer if there was more distinction between what the methods do.

Reply:
I have a change I will push soon to better distinguish between the two.  It's a valid point.  Naming things is always the fun part.